akkeeper-common/src/main/scala/akkeeper/common/config/Configs.scala

```diff
@@ -34,6 +35,14 @@ private[akkeeper] final class AkkeeperConfig(akkeeperConfig: Config) {
       Seq.empty
     }
   }
+
+  lazy val globalResources: Seq[AkkeeperResource] = {
+    if (akkeeperConfig.hasPath("globalResources")) {
+      akkeeperConfig.getConfigList("globalResources").asScala.map(akkeeperResourceFromConfig)
```

Comment:
Can we please name this `global-resources` to be consistent with other property names.

Reply:
Yes, done